doc/cascadia/profiles.schema.json

```diff
@@ -155,263 +159,267 @@
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI black."
-                },
-                "blue": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI blue."
-                },
-                "brightBlack": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI bright black."
-                },
-                "brightBlue": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI bright blue."
-                },
-                "brightCyan": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI bright cyan."
-                },
-                "brightGreen": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI bright green."
-                },
-                "brightPurple": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI bright purple."
-                },
-                "brightRed": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI bright red."
-                },
-                "brightWhite": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI bright white."
-                },
-                "brightYellow": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI bright yellow."
-                },
-                "cyan": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI cyan."
-                },
-                "foreground": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the foreground color of the color table."
-                },
-                "green": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI green."
-                },
-                "purple": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI purple."
-                },
-                "red": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI red."
-                },
-                "white": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI white."
-                },
-                "yellow": {
-                  "$ref": "#/definitions/Color",
-                  "description": "Sets the color used as ANSI yellow."
-                }
+    "Profile": {
```

Comment:
this schema probably says name or guid is required for defaultsettings, eh. i wonder if there's some all_of or any_of magic we can use to make sure that some profiles have guids+settings and some just have settings? don't necessarily block on this ..

Reply:
I already fixed that :)
Profiles in the "defaultSettings" don't have anything required, but ones in ProfileList do

Reply:
Oo, i see